frontend/src/store/modules/shoots/getters.js

```diff
@@ -303,5 +304,10 @@ export default {
     return differenceWith(state.filteredShoots, state.sortedUidsAtFreeze, (filteredShoot, uid) => {
       return filteredShoot.metadata.uid === uid
     }).length
+  },
+  getConditionForType (state, getters, rootState) {
+    return (type) => {
+      return get(rootState.cfg, ['knownConditions', type], getCondition(type))
+    }
```

Comment:
```suggestion
  conditionForType (state, getters, rootState) {
    return type => get(rootState.cfg, ['knownConditions', type], getCondition(type))
  }
```
We never used the prefix `get` in the name of an getter.